app/format/format.tsx

```diff
@@ -113,10 +116,26 @@ export function formatTimestamp(timestamp: { seconds?: number | Long; nanos?: nu
 
 const DATE_RANGE_SEPARATOR = "\u2013";
 
-export function formatDateRange(startDate: Date, endDate: Date) {
-  // TODO: Use `new Intl.DateTimeFormat(...).formatRange` when supported by all browsers.
+export const LAST_N_DAYS_OPTIONS = [7, 30, 90, 180, 365];
+
+export function formatDateRange(startDate: Date, endDate: Date, { now = new Date() } = {}) {
   let startFormat, endFormat;
-  const now = new Date();
+
+  // Special cases for date range picker default options
+  if (isSameDay(now, endDate)) {
+    if (isSameDay(startDate, LOCAL_EPOCH)) {
+      return "All time";
+    }
+    for (const n of LAST_N_DAYS_OPTIONS) {
```

Comment:
Could this be simplified as:
```
if (isSameDay(now, endDate)) {
    if (isSameDay(startDate, LOCAL_EPOCH)) {
      return "All time";
    }
  return `Last ${daysBetween(startDate, endDate)} days`;
}
```

Seems weird for `format.tsx` to have to know about `LAST_N_DAYS_OPTIONS`, and it's probably more readable to say `Last 37 days` than showing the dates.